RR.UnitOfWork/Entities/HRIS/Employee.cs

```diff
@@ -58,6 +59,44 @@ public Employee(EmployeeDto employeeDto, EmployeeTypeDto employeeType)
         Active = employeeDto.Active;
         InactiveReason = employeeDto.InactiveReason;
         EmployeeType = new EmployeeType(employeeType);
+        LinkedIn = employeeDto.LinkedIn;
+        CV = employeeDto.CV;
+        PortfolioLink = employeeDto.PortfolioLink;
+        PortfolioPDF = employeeDto.PortfolioPDF;
+        Referral = employeeDto.Referral;
+        HighestQualification = employeeDto.HighestQualification;
+        School = employeeDto.School;
+        QualificationEndDate = employeeDto.QualificationEndDate;
+        BlackListStatus = employeeDto.BlackListStatus;
+        BlackListReason = employeeDto.BlackListReason;
+        IsCandidate = employeeDto.IsCandidate;
+    }
+
+    public Employee(CandidateDto candidateDto)
+    {
+        Id = candidateDto.Id;
+        Name = candidateDto.Name;
+        Surname = candidateDto.Surname;
+        PersonalEmail = candidateDto.PersonalEmail;
+        Level = candidateDto.PotentialLevel;
```

Comment:
The two constructors have a lot of the same code and can be reduced

Reply:
It's because the one uses the CandidateDto and the other the EmployeeDto